devices/views.py

```diff
@@ -1,6 +1,9 @@
 # coding: utf-8
 from __future__ import unicode_literals
 import datetime
+import time
+from csv import QUOTE_ALL
+from django.utils.translation import ugettext
```

Comment:
convention through out the code is to use:

`from django.utils.translation import ugettext_lazy as _` :)